next.config.js

```diff
@@ -13,8 +13,7 @@ const nextConfig = {
       },
       {
         source: '/stakewars',
-        destination:
```

Comment:
@jackson-harris-iii you can probably revert this entire file. Looks like you don't have any changes here that are intended to be committed, correct?  You appear to have a prettier configuration that is overwriting [this project's config](https://github.com/near/near-discovery/blob/b3c33457c85d9c5792b543c359820e0f9c7161ed/.prettierrc#L3)